oauth2/client-credentials-flow/token-generation/nginx.conf

```diff
@@ -35,7 +35,7 @@ http {
     set_by_lua $deployment 'return os.getenv("THREESCALE_DEPLOYMENT_ENV")';
     set $threescale_backend "https://su1.3scale.net:443";
 
-    location /_threescale/auth {
+    location = /_threescale/check_credentials {
```

Comment:
maybe... I don't want to dictate on the generated ones yet :) 

I have a looooot of suggestions :P